src/Makefile.qt.include

```diff
@@ -234,6 +237,11 @@ RES_ICONS = \
   qt/res/icons/bitcoin.ico \
   qt/res/icons/bitcoin_testnet.ico \
   qt/res/icons/bitcoin.png \
+  qt/res/icons/button_off.png \
+  qt/res/icons/button_on_blue.png \
```

Comment:
I think Checkboxes are appropriate for now.

But we should consider using non-standard UI elements to improve the look and feel. I know, developers hate this. :-)

But have a look at https://github.com/bitcoin/bitcoin/pull/5896. I think people liked it because the look and feel was different then just "Arial 12 and standard UI elements".